test/jdk/sun/security/ssl/SSLCipher/SSLSocketNoServerHelloClientShutdown.java

```diff
@@ -68,7 +71,8 @@ public SSLSocketNoServerHelloClientShutdown() throws Exception {
         super();
         float timeoutFactor = Float.parseFloat(
                 System.getProperty("test.timeout.factor", "1.0"));
-        serverReadTimeout = (int) (5000 * timeoutFactor);
+        serverReadTimeout = (int) (30000 * timeoutFactor);
```

Comment:
My suggestion was to remove the SoTimeout on both the client and server, also including the one now at line 95.

Reply:
This is just a fallback mechanism right now, otherwise the test can hang for a very long time if something goes wrong. If 30 seconds is not enough for the test to run then we fail.

Reply:
I still think two sync mechs (one to start the server read and one to start the client shutdown) would be more deterministic/reliable, but I'll approve if you really want to keep it this way.